### PR TITLE
EFRS-808: Review Readme.md file

### PR DESCRIPTION
**Pull request by andreigaevsky**

### CODE REVIEW

**Comment by pospielov** [comment]
One general thing - please remove every "model" information and put "Faces Collection"

File: README.md [hunk 1]
```diff
@@ -229,16 +228,14 @@ Response body on success:
 
 
 
-### **Recognize a face**
+### Recognize faces from given image
 
 Recognizes faces from the uploaded images.
 ```http request
-curl  -X POST "http://localhost:8000/api/v1/recognize" \
+curl  -X POST "http://localhost:8000/api/v1/recognize?limit=<limit>&prediction_count=<prediction_count>" \
```
[the remark this answers is not in the document]
Comment (andreigaevsky) [reply README.md:235]: Nope. Now it's a constant value `0.5D`

File: README.md [hunk 2]
```diff
@@ -378,6 +377,54 @@ Response body on success:
 
 
 
+### Verify faces from given image
+
+Recognizes faces from the uploaded images.
+```http request
+curl  -X POST "http://localhost:8000/api/v1/faces/<image_id>/verify?limit=<limit>" \
```
[the remark this answers is not in the document]
Comment (andreigaevsky) [reply README.md:384]: Nope. Now it's a constant value `0.5D`

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pospielov) [reply README.md:235]: But why?) how we chose the value?

[the remark this answers is not in the document]
Comment (andreigaevsky) [reply README.md:235]: It's not my decision, but Maryna's. I think, we should add threshold in an another task.

File: README.md [hunk 3]
```diff
@@ -198,23 +200,20 @@ By using the created API key, the user can add an image as an example of the fac
 Creates an example of the face by saving images. To train the system, you can add as many images as you want.
 
 ```http request
-curl  -X POST "http://localhost:8000/api/v1/faces/?subject=<face_name>" \
+curl  -X POST "http://localhost:8000/api/v1/faces?subject=<face_name>&det_prob_threshold=<det_prob_threshold>" \
 -H "Content-Type: multipart/form-data" \
 -H "x-api-key: <model_api_key>" \
```
Comment (pospielov) [inline README.md:205]: Everywhere
```suggestion
-H "x-api-key: <faces_collection_api_key>" \
```

File: README.md [hunk 4]
```diff
@@ -378,6 +377,54 @@ Response body on success:
 
 
 
+### Verify faces from given image
+
+Compares faces from the uploaded images with provided face id.
```
Comment (pospielov) [inline README.md:382]:
```suggestion
Compares faces from the uploaded image with face in saved image id.
```

File: README.md [hunk 5]
```diff
@@ -198,23 +200,20 @@ By using the created API key, the user can add an image as an example of the fac
 Creates an example of the face by saving images. To train the system, you can add as many images as you want.
 
 ```http request
-curl  -X POST "http://localhost:8000/api/v1/faces/?subject=<face_name>" \
```
Comment (pospielov) [inline README.md:201]: Let's rename all `face_name` to `subject` and `face_id` to `image_id`

File: README.md [hunk 6]
```diff
@@ -229,26 +228,24 @@ Response body on success:
 
 
 
-### **Recognize a face**
+### Recognize faces from given image
 
 Recognizes faces from the uploaded images.
```
Comment (pospielov) [inline README.md:233]:
```suggestion
Recognizes faces from the uploaded image.
```

**Review by pospielov — APPROVED** [review]

**Pull request merged**